packages/framework/src/Controller/Admin/DefaultController.php

```diff
@@ -147,4 +179,20 @@ protected function addWarningMessagesOnDashboard(): void
             );
         }
     }
+
+    /**
+     * @param int $previous
+     * @param int $current
+     * @return int
```

Comment:
why not to round it in the twig ?

Reply:
Well I can typehint it this way and by design there is no expectation of such precision.